resources/metadata/mavenStaticCodeChecks.yaml

```diff
@@ -26,6 +26,15 @@ spec:
           - PARAMETERS
           - STAGES
           - STEPS
+      - name: m2Path
+        description: Path to the local m2 directory.
+        type: string
+        default: '~/.m2/'
```

Comment:
Do we need a default here? I think the maven default should be fine. On windows '~/.m2/' would even be wrong...